index.yaml

```diff
@@ -16544,27 +16589,6 @@ entries:
     - assets/rancher-compliance-crd/rancher-compliance-crd-107.0.0+up1.0.0-rc.5.tgz
     version: 107.0.0+up1.0.0-rc.5
   rancher-csp-adapter:
-  - annotations:
```

Comment:
Was this intentional or is the side-effect of `autobump`?

Reply:
Nvm, it seems that the unrc [PR](https://github.com/rancher/charts/pull/5977) for the csp-adapter didn't update the index.yaml.